MCPClient/Sources/stdioTransport/DataChannel+StdioProcess.swift

```diff
@@ -143,17 +147,19 @@ extension DataChannel {
       if stdout.fileHandleForReading.fileDescriptor != stderr.fileHandleForReading.fileDescriptor {
         Task {
           for await data in stderr.fileHandleForReading.dataStream {
-            logger.log("Received error:\n\(String(data: data, encoding: .utf8) ?? "nil")")
+            if verbose {
+              logger.log("Received error:\n\(String(data: data, encoding: .utf8) ?? "nil")")
+            }
```

Comment:
nit: this is from my change, would be better to revert
```suggestion
            logger.log("Received error:\n\(String(data: data, encoding: .utf8) ?? "nil")")
```